web/.env.example

```
@@ -39,6 +39,29 @@ NEXT_PUBLIC_STACK_PROJECT_ID=
 NEXT_PUBLIC_STACK_PUBLISHABLE_CLIENT_KEY=
 STACK_SECRET_SERVER_KEY=
 
+# Personal-account Iroh trust broker. The Iroh Services project API key is not
+# accepted by this process; it belongs only in the isolated Rust relay minter.
+# Grant verification JSON maps the current and previous KIDs to Ed25519 SPKI
+# PEM strings. The developer binding override remains off unless all three
+# override settings explicitly match the authenticated user and deployment.
+CMUX_IROH_LAN_DISCOVERY_SECRET_B64=
+# Independent 32-byte base64 secret used only to derive opaque account subjects
+# for short-lived offline endpoint attestations.
+CMUX_IROH_ACCOUNT_SUBJECT_SECRET_B64=
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The CMUX_IROH_ACCOUNT_SUBJECT_SECRET_B64 key should go before the CMUX_IROH_LAN_DISCOVERY_SECRET_B64 key

(UnorderedKey)